casas_gis_old/casas/scripts/usa

```diff
@@ -40,7 +40,7 @@
 #%Module
 #%  description: Map the output of CASAS models for the conterminous United States, Mexico, and Central America
 #%  keywords: models
-#%End 
+#%End
```

Comment:
Not sure what is being done here: removed and then added the same line?

Reply:
Just an empty space after the `d` of `End` was deleted.